examples/fileapi-express_4/server.js

```diff
@@ -101,6 +102,6 @@ fs.mkdir(path.join(__dirname, '/uploaded'), function () {
   // ignore errors, most likely means directory exists
   console.log('Uploaded files will be saved to %s/uploaded.', __dirname);
   console.log('Remember to wipe this directory if you upload lots and lots.');
-  app.listen(8080);
+  server.listen(8080);
```

Comment:
Can you add a callback for the `listening` event and move the next line (`console.log()`) there?